### PR TITLE
feat: adding home page

### PR DESCRIPTION
**Pull request by cindy-sf**

### CODE REVIEW

File: src/views/Home/components/ArticleCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { radius, spaces } from '@src/styles/theme'
+
+import Image from '@components/Image'
+import Text from '@components/Text'
+
+import type { News } from '../../types'
```
Comment (j0nathansf) [inline src/views/Home/components/ArticleCard/index.tsx:9]: Maybe we should add an alias

File: src/views/Home/components/ArticleCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+const ArticleCardWrapper = styled.View`
+  margin-right: ${spaces.X_LARGE}px;
+  margin-top: ${spaces.SMALL}px;
+`
+
+const ArticleCardContent = styled.View`
+  width: 260px;
+  height: 155px;
```
Comment (j0nathansf) [inline src/views/Home/components/ArticleCard/index.tsx:18]: These values are not present in our style files ?

File: src/views/Home/components/SpotlightCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { View } from 'react-native'
+
+import Text from '@components/Text'
+import Image from '@components/Image'
+
+import { radius, spaces } from '@src/styles/theme'
+
+import type { News } from '../../types'
```
Comment (j0nathansf) [inline src/views/Home/components/SpotlightCard/index.tsx:10]: Same

File: src/views/Home/components/SpotlightCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      marginBottom: spaces.LARGE,
+    }}
+  >
+    <View
+      style={{
+        borderRadius: radius.LARGE,
+        overflow: 'hidden',
+        width: 325,
+        marginBottom: spaces.SMALL,
+      }}
```
Comment (j0nathansf) [inline src/views/Home/components/SpotlightCard/index.tsx]: Styled components 🤷‍♂️

File: src/views/Home/index.tsx [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  const [shouldDisplayError, setShouldDisplayError] = useState<boolean>(false)
+  const [isDataFetching, setIsDataFetching] = useState<boolean>(false)
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        setIsDataFetching(true)
+        // eslint-disable-next-line no-undef
+        const response = await fetch(
+          'https://api.betaseries.com/news/last?key=72ede40de1b2&limit=15'
```
Comment (j0nathansf) [inline src/views/Home/index.tsx]: Isn't there a way to fix this eslint error?

File: src/views/Home/types.ts [hunk 6]
```diff
@@ -0,0 +1,8 @@
+export interface News {
+  date: Date
+  id: number
+  // eslint-disable-next-line camelcase
+  picture_url: string
```
Comment (j0nathansf) [inline src/views/Home/types.ts]: Same for this warning

File: src/views/Landing/index.tsx [hunk 7]
```diff
@@ -9,7 +9,7 @@ import { landingInfos } from './constants'
 import StepIndicator from './components/StepIndicator'
 import LandingInfos from './components/LandingInfos'
 
-const Landing = () => {
+const Landing = ({ navigation }: { navigation: any }) => {
```
Comment (j0nathansf) [inline src/views/Landing/index.tsx]: Avoid any

File: src/views/Landing/index.tsx [hunk 8]
```diff
@@ -26,10 +26,18 @@ const Landing = () => {
     { useNativeDriver: false }
   )
 
+  const goToHomeScreen = (): void => {
+    // navigation.reset({
+    // index: 0,
+    // routes: [{ name: 'Home' }],
+    // })
```
Comment (j0nathansf) [inline src/views/Landing/index.tsx]: Dead code
Don't commit

**Review by j0nathansf — CHANGES_REQUESTED** [review]
Lots of things to review

File: src/views/Home/index.tsx (hunk 5 above)
Comment (j0nathansf) [reply src/views/Home/index.tsx]: Avoid putting the key in hard
Create a file that contains this kind of data (which you will obviously add in the .gitignore file)

File: src/views/Home/components/ArticleCard/index.tsx (hunk 2 above)
Comment (cindy-sf) [reply src/views/Home/components/ArticleCard/index.tsx:18]: Nop

File: src/views/Home/components/ArticleCard/index.tsx (hunk 1 above)
Comment (cindy-sf) [reply src/views/Home/components/ArticleCard/index.tsx:9]: That's not necessary

**Review by j0nathansf — APPROVED** [review]
gg 💯

**Pull request merged**